src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
+    private int top;
+    private int bottom;
+    private int side;
+
+    public IsoscelesTrapezoid(String color, int top, int bottom, int side) {
+        super(color);
+        this.side = side;
+        this.bottom = bottom;
+        this.top = top;
+    }
+
+    @Override
```

Comment:
The method draw() should not begin with an empty line. Please remove the empty line before the method declaration.